internal/context/init.go

```diff
@@ -63,11 +63,11 @@ func Init(projectName string, outDir string) *projectconfig.ZeroProjectConfig {
 }
 
 // loadAllModules takes a list of module sources, downloads those modules, and parses their config
-func loadAllModules(moduleSources []string) map[string]moduleconfig.ModuleConfig {
+func loadAllModules(moduleSources projectconfig.Modules) map[string]moduleconfig.ModuleConfig {
```

Comment:
This shouldn't accept projectconfig.Modules, just a slice of string. At this point there is no project config and all we have are some repo paths.

Reply:
changed it back to a string

Reply:
Thanks, could you remove the changes to the registry section as well?